reviews/templatetags/documents_list.py

```diff
@@ -222,6 +223,10 @@ def documents_list(review, user):
 
     pdf_container.items.append(proposal_pdf)
 
+    if is_secretary(user):
+        pdf_container.date_start_edit_link = reverse('proposals:update_date_start',
+                                                     args=[proposal.pk])
```

Comment:
I don't really like adding this link to the document list code. You're not updating documents. 

This should be a review action instead, I think.

Reply:
I agree that does make more sense! fixed